rdr_service/genomic/genomic_job_components.py

```diff
@@ -2691,6 +2697,7 @@ def _get_long_read_participants(self, limit=None):
                 Code.value == 'WhatRaceEthnicity_Black',
                 GenomicSetMember.genomeType.in_(['aou_wgs']),
                 GenomicSetMember.genomicWorkflowState != GenomicWorkflowState.IGNORE,
+                or_(GenomicGCValidationMetrics.contamination <= 0.01, GenomicGCValidationMetrics.contamination == ""),
```

Comment:
We don't want to include `contamination == ""`. This means that the GC did not provide contamination data which means that we don't know if the sample is contaminated or not. We also want to include a filter for `GenomicGCValidationMetrics.ignore_flag == 0`. I should have mentioned both of these conditions earlier since we are now joining to `GenomicGCValidationMetrics`.

Reply:
Sure np, Ill remove/add it.

Reply:
aeb52fd98db9efa1c7da2b937eb28dc6ee4e33a3